pages/edit/[id].js

```diff
@@ -28,6 +28,10 @@ export default function EditPage() {
   if (project === " " || project === undefined) {
     return <h2>loading...</h2>;
   } else {
-    return <EditForm project={project} />;
+    return (
+      <>
+        <EditForm project={project} />
+      </>
```

Comment:
Why did you add the fragments here? I don't see any need for them.

Reply:
yapp. you're right...it's a left over...i have removed it